### DISCUSSION TITLE
[Bug report] Premium Themes shipped with too little content

### DISCUSSION THREAD

**Discussion by FuchsiaArt** [GetPublii/Publii#1496] [Bug Report]
### Operating system

Windows 10

### Publii version

0.45.2

### Issue type

Something else

### Bug description

A great thanks for providing us with such a nice Publii software for designing **amazing websites**. 

When I purchased the **Melissa V 2.1.0.0** as a premium product, I was expecting to having more examples in the designed template for example such that it includes examples **(a)** Typography, **(b)** Pages, **(c)** Galleries etc.. to get us started quickly. But to our astonishment the themes was shipped with the bear minimum. Is this because we selected the wrong theme?

Kind regards,
Lihui Lee

**Comment by candidexmedia** [comment]
Hi @FuchsiaArt : This is normal: All the themes come with no content, as content is stored in a sqlite database that's unique to each site, and not the theme download.

My opinion is that although it would be good to have some content to get you started, the flip side of that is being stuck with a bunch of content that you have to manually delete, and you might forget a few default elements that pop up on your actual site. It also makes switching between themes easier.

My advice: Explore the user docs, and explore every option, page, menu item, editor, etc. and if you get stuck, don't hesitate to ask us questions.

**Reply by FuchsiaArt** [reply]
A great thanks for taking your time for answering my query and for offering further support if required. 

Your **Publii** software is no doubt the best software around for non-specialist . From my opinion for helping to choose the right **theme** it will be helpful to present _all the sample websites_ with the same minimal pictures and default layout which will make comparison easier.

Then for downloading the theme, the user can specify to download all the simplified demo website or the “empty” website. 

From my opinion having the full simplified website would help beginners to make a quicker start.

Than in a separate link one can show as an example the more elaborate website.

**Reply by candidexmedia** [reply]
Those are great suggestions @FuchsiaArt ! Hopefully, the devs take this into account (just to clarify: I don't work for Publii, I just make themes independently)
